set-balance.md

```diff
@@ -81,15 +81,112 @@ A `Result` is the return value of an execution step.
  // ---------------------------------------------------------
 ```
 
+Public Getters
+--------------
+
+### `total_balance`
+
+Retrieves the total balance of an account.  This includes both the free and
+reserved balances.
+
+```k
+    syntax Int ::= "total_balance" "(" AccountId ")" [function, functional]
+ // -----------------------------------------------------------------------
+    rule [[ total_balance(WHO) => FREE_BALANCE +Int RESERVED_BALANCE ]]
+         <account>
+           <accountID> WHO </accountID>
+           <freeBalance> FREE_BALANCE </freeBalance>
+           <reservedBalance> RESERVED_BALANCE </reservedBalance>
+           ...
+         </account>
+```
+
+### `can_slash`
+
+Determines if an account’s free balance is over the value provided.  This is
+often used to determine if an account has enough balance to cover a potential
+slash, hence the name.
+
+```k
+    syntax Bool ::= "can_slash" "(" AccountId "," Int ")" [function, functional]
+ // ----------------------------------------------------------------------------
+    rule [[ can_slash(WHO, AMOUNT) => FREE_BALANCE >=Int AMOUNT ]]
+         <account>
+           <accountID> WHO </accountID>
+           <freeBalance> FREE_BALANCE </freeBalance>
+           ...
+         </account>
+```
+
+### `total_issuance`
+
+Retrieves the total outstanding amount of currency outstanding.  This will
+always be equal to the sum of all free and reserved balances in all active
+accounts, except when the balances module is executing.
+
+```k
+    syntax Int ::= "total_issuance" [function, functional]
+ // ------------------------------------------------------
+    rule [[ total_issuance => TOTAL_ISSUANCE ]]
+         <totalIssuance> TOTAL_ISSUANCE </totalIssuance>
+```
+
+### `minimum_balance`
+
+Retrieves the minimum balance, also known as the existential deposit.
+
+Other than when this module is executing, no account will ever have a balance
+strictly between this and zero.
+
+```k
+    syntax Int ::= "minimum_balance" [function, functional]
+ // -------------------------------------------------------
+    rule [[ minimum_balance => EXISTENTIAL_DEPOSIT ]]
+         <totalIssuance> EXISTENTIAL_DEPOSIT </totalIssuance>
```

Comment:
Should this be the `<existentialDeposit>` cell?

Reply:
Indeed it should, thanks!